Docs/FHS Bundles.md

```diff
@@ -0,0 +1,123 @@
+
+```
+share/MyApp.resources
+bin/MyApp
+```
+
+or:
+
+```
+share/MyFramework.resources
+lib64/libMyFramework.so
+```
+
+The bundle path for an installed bundle is the same as its resources path. That's the path that needs to be passed into `CFBundleCreate()` and equivalents for the bundle to be successfully created; passing associated directory names will return `NULL`. [⊗](#nullForInnerPaths)
+
+As an example, the following values are produced by invoking these functions on the bundle created with `CFBundleCreate(NULL, …(…"/usr/local/share/MyFramework.resources"))`:
+
+Function | Path returned
+---|---
+`CFBundleCopyBundleURL` | `/usr/local/share/MyFramework.resources`
+`CFBundleCopyExecutableURL` | `/usr/local/lib/libMyFramework.so`
+`CFBundleCopyResourcesDirectoryURL` | `/usr/local/share/MyFramework.resources`
+`CFBundleCopySharedSupportURL` |`/usr/local/share/MyFramework.resources/SharedSupport`
+`CFBundleCopyPrivateFrameworksURL` | `/usr/local/libexec/MyFramework.executables/Frameworks`
+`CFBundleCopySharedFrameworksURL` | `/usr/local/libexec/MyFramework.executables/SharedFrameworks`
+`CFBundleCopyBuiltInPlugInsURL` | `/usr/local/libexec/MyFramework.executables/PlugIns`
+`CFBundleCopyAuxiliaryExecutableURL(…, CFSTR("myexec"))` | `/usr/local/libexec/MyFramework.executables/myexec`
+`CFBundleCopyResourceURL(…, CFSTR("Welcome"), CFSTR("txt") …)` | `/usr/local/share/MyFramework.resources/en.lproj/Welcome.txt`
+
+The structure inside any of these paths is the same as that of an iOS bundle containing the appropriate subset of files for its location; for example, the resources folder contains localization `.lproj` subdirectories, the `Frameworks` directory inside the `….executables` contains frameworks, and so on.
+
+## Freestanding bundles
+
+We will also support freestanding bundles on platforms that also support installed bundles. These bundles are structured much like their Windows counterparts, where there is a binary and a `.resources` directory on the side, like so:
+
+```
+./MyFramework.resources
+./libMyFramework.so
+```
+
+### Bundle Paths
+
+The bundle path for an installed bundle is the same as its resources path. That's the path that needs to be passed into `CFBundleCreate()` and equivalents for the bundle to be successfully created; passing associated paths, including the path to the executable, will return `NULL`. [⊗](#nullForInnerPaths)
+
+The `.resources` directory functions exactly like an iOS bundle, returning the same paths. By way of example, for a freestanding bundle created with `CFBundleCreate(NULL, …(…"./MyFramework.resources"))`:
+
+Function | Path returned
+---|---
+`CFBundleCopyBundleURL` | `…/MyFramework.resources`
```

Comment:
Just to be sure: does `...` mean the parent directory?

Reply:
It meant 'the same directory as … in the other entries', but I see how it can be confusing. I will clarify this.